lib/happo/public/HappoDiffs.jsx

```diff
@@ -94,21 +100,91 @@ HappoNewImages.propTypes = {
   images: PropTypes.arrayOf(PropTypes.shape(imageShape)),
 };
 
+const Swiper = React.createClass({
+  propTypes: {
+    previous: PropTypes.string.isRequired,
+    current: PropTypes.string.isRequired,
+  },
+
+  getInitialState() {
+    return {
+      cursorLeftPercent: 50, // Start in the center
+    };
+  },
+
+  _handleMouseMove(event) {
+    const leftPx = event.pageX - event.target.offsetLeft;
+    const leftPercent = leftPx / event.target.offsetWidth * 100;
+
+    console.log(event.target)
+    console.log(leftPx, event.target.offsetWidth, leftPercent);
+    this.setState({
+      cursorLeftPercent: leftPercent,
+    })
+  },
+
+  render() {
+    const {
+      previous,
+      current,
+    } = this.props;
+
+    const { cursorLeftPercent } = this.state;
+    const clipPath =
+      `polygon(${cursorLeftPercent}% 0, 100% 0, 100% 100%, ${cursorLeftPercent}% 100%)`;
+
+    return (
+      <div
+        className='happo-swiper'
+        style={{
+          backgroundImage: `url(${previous})`,
```

Comment:
If the previous image is bigger than the current, how will this look? It might be good to modify some of the example images to showcase this scenario (and the reverse).